luarules/gadgets/scav_spawner_defense.lua

```diff
@@ -155,6 +155,14 @@ if gadgetHandler:IsSyncedCode() then
 	--FrontbusterPopulation = 0
 	HumanTechLevel = 0
 
+	--dynamic difficulty stuff
+	local dynDifficulty0to1 -- ranges from 0-1 based on if the scavengers last wave compared against all non-scav/raptor team powers. 1/5th = 0, 1/3rd = 1 with a linear progression between
+	local dynDifficultyClamped -- ranges from .85 to 1.05 based on dynDifficulty0to1 with a linear progression in between.
```

Comment:
```suggestion
	local dynamicDifficulty
	local dynamicDifficultyClamped
```
Short forms and abbreviations should be avoided in code, implementation details should be in comments, as it is easy for the comment to become out of date and provide wrong information.